clang/lib/Lex/PPMacroExpansion.cpp

```diff
@@ -1797,55 +1798,62 @@ void Preprocessor::ExpandBuiltinMacro(Token &Tok) {
                                            diag::err_feature_check_malformed);
         return II && HasExtension(*this, II->getName());
       });
-  } else if (II == Ident__has_builtin) {
```

Comment:
There seem to be irrelevant formatting changes, which should be avoided.

Reply:
> I heard in the past we kind of do fix-as-you-go with clang-format where sometimes stuff like this happens.

Sort of. We fix the lines that are touched, but don't reformat otherwise untouched lines because that makes review harder now and blames harder in the future. A better approach is to apply clang-format to the range of code you care about as an NFC change. That doesn't require review. Then you apply your changes on top of the now-formatted code.

Reply:
Thanks Aaron, I'll do that in the future. 
Yaxun marked this issue as resolved I'm going to assume I don't need to take any action this time, but I'm happy to if others prefer it.

Reply:
I'd prefer it; it makes it easier to see what's actually changed. But if that's a PITA, I won't insist.

Reply:
Shouldn't be too bad, will try it.

Reply:
I think the latest commit does the right thing

Reply:
@AaronBallman Yeah here the clang-format CI [failed](https://github.com/llvm/llvm-project/actions/runs/13249258371/job/36983046889?pr=126324) telling me I need to update unrelated code. Should I just ignore it?

Reply:
thanks!